roles/cifmw_cephadm/tasks/pre.yml

```diff
@@ -158,3 +158,49 @@
     mode: "0644"
     config_overrides: "{{ cifmw_cephadm_conf_overrides }}"
     config_type: ini
+
+- name: Check the list of services
```

Comment:
Can we please put lines 162 to 206 in a separate file like `k8s_openstack_object_config.yml` and then in pre.yml include that tasks file? The inclusion of that tasks file should be wrapped in a `when: cifmw_openshift_kubeconfig is defined`.